MostMinimalWebFramework.py

```diff
@@ -23,26 +24,22 @@ class Response:
     content_type: str = "text/html"
 
 
-class JSONResponse:
-    def __new__(cls, *args, **kwargs):
-        return Response(content_type="application/json", *args, **kwargs)
+def jsonify(*args, **kwargs):
+    return Response(content_type="application/json", *args, **kwargs)
```

Comment:
Sorry, I don't like the idea of Response is class and Json response is function. IMHO, It decreases readability of implemented views.